test/schema/export_test.js

```diff
@@ -21,14 +24,27 @@ describe('#exportToSchemasUniqueCases()', () => {
     let schemas = exportToSchemas([ns]);
     expect(schemas).to.have.length(16);
     expect(schemas).to.eql(expectedSchemas);
+    expect(validateSchema(schemas));
   });
 });
 
 function exportNamespaces(...namespace) {
   let schemas = exportToSchemas(namespace);
+  expect(validateSchema(schemas));
   return schemas;
 }
 
 function importFixture(name, ext='.json') {
   return JSON.parse(fs.readFileSync(`${__dirname}/fixtures/${name}${ext}`, 'utf8'));
 }
+
+function validateSchema(schema) {
+    const v = new validator();
+    const extendedSchema = importSchema();
+    return v.validate(schema, extendedSchema);
```

Comment:
So, in essence, what jsonschema is doing is just making sure json is validated against a schema. And that's all. We should search for a new library or decide to implement our own, and maybe prioritize that against other tasks.